src/devices_models/devices/common/range_constraint.jl

```diff
@@ -150,7 +150,7 @@ function device_semicontinuousrange!(
     return
 end
 
-#This function looks suspicious and repetitive. Needs verification
+# This function looks suspicious and repetitive. Needs verification
```

Comment:
Is this comment still valid?

Reply:
Probably not as originally intended but when I refactor range constraints I'll remove this comment. This will be resolved in a future PR.